### PR TITLE
Suspense

### PR DESCRIPTION
**Pull request by bryan-robitaille**
# Summary | Résumé

This PR introduces a new component that can be used as a Loading / Busy spinner.  It leverages the [react-spinners](https://github.com/davidhu2000/react-spinners) library.  The spinner is currently set on the Submit form page while the form validates and connects to the lambda to handle the form submission.

A11y - Tested in Mac OS Voice Over.  It was awesome.

| French Spinner | English Spinner|
|:----:|:----:|
|french_suspense|english_suspense|

### CODE REVIEW

**Comment by brdunfield** [comment]
Do you think we could also add this to the new admin pages? Particularly upload and settings when the form json is uploaded?

File: components/globals/Loader.tsx [hunk 1]
```diff
@@ -0,0 +1,23 @@
+import React from "react";
+import { css } from "@emotion/react";
+import Loader from "react-spinners/PacmanLoader";
```
Comment (brdunfield) [inline components/globals/Loader.tsx:3]: As much as I love it, I feel like the PacmanLoader might not be the _best_ choice.. 🤔

Comment (bryan-robitaille) [reply components/globals/Loader.tsx:3]: lol I know... It's more of a fun placeholder until design provides their input.  I can also swap it out for something more 'standard'.

**Comment by brdunfield** [comment]
> > Do you think we could also add this to the new admin pages? Particularly upload and settings when the form json is uploaded?
> 
> Definitely I can amend this PR. Wanted to check in with you first @brdunfield because I know you were modding those pages for state updates as well.

Yep! All my changes should have been in: https://github.com/cds-snc/platform-forms-client/pull/203
which was merged yesterday!

**Review by brdunfield — APPROVED** [review]
LGTM! 🦑

**Pull request merged**